README.md

```diff
@@ -22,19 +22,15 @@ For more information, please check our [documentation on colin.readthedocs.io](h
 
 ## Usage
 
-### How to test colin locally
+### How to test Colin locally
 
 ```bash
-    make check-local -e TEST_IMAGE_NAME=<image_name> -e ANSIBLE_EXTRA_ARGS=-vv -e CONFIG=fedora -e ARTIFACTS_DIR=<directory_for_results> -e RESULTS=result.json  -e setup=true
+    make check-local -e TEST_IMAGE_NAME=<image_name> -e ANSIBLE_EXTRA_ARGS=-vv -e CONFIG=<config_file> -e ARTIFACTS_DIR=<directory_for_results> -e RESULTS=<result_file>  -e setup=true
 ```
 
-which runs ansible playbook, by a command:
-
-```bash
-    ansible-playbook $(ANSIBLE_EXTRA_ARGS) -e config=$(CONFIG) -e subject=$(TEST_IMAGE_NAME) -e results=$(RESULTS) -e artifacts_dir=$(ARTIFACTS_DIR) ./local.yml -e setup=true
-```
-The parameters used in ansible specifies:
+The parameters used in command specify:
 - TEST_IMAGE_NAME ... name of the image which colin tests
+- ANSIBLE_EXTRA_ARGS ... extra arguments for ansible command
 - CONFIG ... name of default configuration file which is being used. By default `fedora`
 - ARTIFACTS_DIR ... directory where the results are stored. The directory has to already exist. By default `./artifacts`
```

Comment:
This is probably not true. The directory has to exist for the colin,  but not for the ansible playbook which creates it if needed.